docs/en/seatunnel-engine/checkpoint-storage.md

```diff
@@ -186,3 +186,40 @@ seatunnel:
 
 ```
 
+### Enable cache
+
+When storage:type is hdfs, cache is disabled by default. If you want to enable it, set `disable.cache: false`
```

Comment:
Could you share what risks will be caused if cache is turned on?

Reply:
> Could you share what risks will be caused if cache is turned on?

I think in the scenario of seatunel, it should not be turned on at any time.

Reply:
> Could you share what risks will be caused if cache is turned on?

You can take a look at the bug #6678 I proposed, which is [S3 connector], which causes occasional task failures due to the use of cache. at the same time, you can also refer to these two issues, which are bug: https://issues.apache.org/jira/browse/HADOOP-15819 and https://github.com/aws/aws-sdk-java/issues/2337 about hadoop-aws and aws-sdk-java. The problems caused by turning on cache in multithreaded environment are described in detail.

Reply:
make sense to me.